examples/amm/README.md

```diff
@@ -101,7 +101,8 @@ Type each of these in the GraphiQL interface and substitute the env variables wi
 
 To properly setup the tokens in the proper chains, we need to do some transfer operations:
 
-- Transfer 50 FUN1 from `$OWNER_AMM` in `$CHAIN_AMM` to `$OWNER_1` in `$CHAIN_1`, so they're in the proper chain
+- Transfer 50 FUN1 from `$OWNER_AMM` in `$CHAIN_AMM` to `$OWNER_1` in `$CHAIN_1`, so they're in the proper chain.
+  Navigate to `http://localhost:8080/chains/$CHAIN_AMM/applications/$FUN1_APP_ID` and enter:
```

Comment:
Let's remind the reader to replace the bash variables though. Alternatively, we are echoing URLs in a bash section in several examples already.